exonum/src/blockchain/schema.rs

```diff
@@ -19,6 +19,31 @@ use storage::{Fork, ListIndex, MapIndex, MapProof, ProofListIndex, ProofMapIndex
 use helpers::Height;
 use super::{Block, BlockProof, Blockchain};
 use super::config::StoredConfiguration;
+use super::transaction::ExecutionStatus;
+
+/// Defines `&str` constants with given name and value.
```

Comment:
Nitpick: This comment is technically wrong; the name is prefixed with the `"core."` string.